docs/reference/migration/migrate_8_0.asciidoc

```diff
@@ -92,3 +93,4 @@ include::migrate_8_0/settings.asciidoc[]
 include::migrate_8_0/snapshots.asciidoc[]
 include::migrate_8_0/threadpool.asciidoc[]
 include::migrate_8_0/transport.asciidoc[]
+include::migrate_8_0/java_time.asciidoc[]
```

Comment:
If you move the breaking change under mapping changes, I'd still leave this file include here.

Reply:
why would we need `java_time.asciidoc` file if I move it to mapping changes file?